src/rpc/rawtransaction.cpp

```diff
@@ -582,11 +582,16 @@ UniValue decodescript(const JSONRPCRequest& request)
 }
 
 /** Pushes a JSON object for script verification or signing errors to vErrorsRet. */
-static void TxInErrorToJSON(const CTxIn& txin, UniValue& vErrorsRet, const std::string& strMessage)
+static void TxInErrorToJSON(const CTxIn& txin, const CScriptWitness& witness, UniValue& vErrorsRet, const std::string& strMessage)
 {
     UniValue entry(UniValue::VOBJ);
     entry.push_back(Pair("txid", txin.prevout.hash.ToString()));
     entry.push_back(Pair("vout", (uint64_t)txin.prevout.n));
+    UniValue txinwitness(UniValue::VARR);
+    for (unsigned int i = 0; i < witness.stack.size(); i++) {
+        txinwitness.push_back(HexStr(witness.stack[i].begin(), witness.stack[i].end()));
+    }
+    entry.push_back(Pair("txinwitness", txinwitness));
```

Comment:
"txinwitness" here seems derived from the variable name, in the context of the entry object it is understood the entry represents a transaction input error, using "witness" instead matches the other entries closer. That is, the error has a "sequence" not a "txinnsequence" or similar.